tests/testDefect.py

```diff
@@ -31,10 +32,8 @@
 import lsst.afw.display.utils as displayUtils
 import lsst.ip.isr as ipIsr
 
-try:
-    type(display)
-except NameError:
-    display = False
+# set to True to display images in ds9
+display = False
```

Comment:
I have no idea, but it's pretty strange. We don't have a good unified way to set a "display things in tests" variable. lsstDebug is kind of like that, but there are some tickets to rethink that system.